pages/wiki/Locale.md

```diff
@@ -1,109 +1,138 @@
+| `ga_IE`  | Irish                           |
+| `gv_IM`  | Manx                            |
+| `he`     | Hebrew                          |
+| `hi_IN`  | Hindi                           |
+| `hr`     | Croatian                        |
+| `hu`     | Hungarian                       |
+| `is_IS`  | Icelandic                       |
+| `it`     | Italian                         |
+| `ja`     | Japanese                        |
+| `ko`     | Korean                          |
+| `lt`     | Lithuanian                      |
+| `lv_LV`  | Lithuanian                      |
+| `nl`     | Dutch                           |
+| `no`     | Norwegian                       |
+| `pl`     | Polish                          |
+| `pt`     | Portuguese                      |
+| `pt_BR`  | Portuguese (Brazil)             |
+| `ro`     | Romanian                        |
+| `ru`     | Russian                         |
+| `si_LK`  | Sinhala                         |
+| `sk`     | Slovak                          |
+| `sr_CS`  | Serbian                         |
+| `sv`     | Swedish                         |
+| `th`     | Thai                            |
+| `tr`     | Turkish                         |
+| `uk`     | Ukrainian                       |
+| `vi`     | Vietnamese                      |
+| `zh`     | Simplified Chinese              |
+| `zh_HK`  | Traditional Chinese (Hong Kong) |
+| `zh_TW`  | Traditional Chinese             |
+<!-- | `lol_US` | LOLCAT                          | -->
 
 ## Customising messages
 
-Since all of the plugins' messages are defined in the messages file, you can fully customise the look and feel of your server by changing the text and colour scheme for all EssentialsX commands.
+In addition to using EssentialsX's built-in translations, you can also customise the text, appearance and functionality of EssentialsX messages!
 
-Simply place your edited locale file (with appropriate locale suffix) in the `plugins/Essentials` folder and make sure the locale setting in the config file matches (`locale: en` -> `messages_en.properties`), then restart the server.
+### Changing the default colours
 
-If you are having trouble, [try this tutorial video](https://www.youtube.com/watch?v=9Uan2oJ9zK8). Note that this video is for the original Essentials plugin, not EssentialsX.
+**NEW IN 2.21.0:** If you just want to change the colours used in the standard EssentialsX messages, you can do so using the `message-colors` section of `config.yml`:
 
-You may need to update the file on occasion to deal with changes in EssentialsX such as parameter changes or changes to the actual syntax of the messages. This can be done with a diff tool like [Diffchecker](https://www.diffchecker.com/), which highlights changes between two text files.
+```yaml
+# Change the default primary and secondary colours used in EssentialsX messages.
+# Some messages may use custom colours, which will need to be edited in the appropriate message files.
```

Comment:
🦅
```suggestion
# Change the default primary and secondary colors used in EssentialsX messages.
# Some messages may use custom colors, which will need to be edited in the appropriate message files.
```

Reply:
This is the same wording as in the config